sdcm/nemesis.py

```diff
@@ -1298,7 +1302,9 @@ def _add_and_init_new_cluster_nodes(self, count, timeout=MAX_TIME_WAIT_FOR_NEW_N
             raise
         for new_node in new_nodes:
             new_node.wait_native_transport()
-        self.cluster.wait_for_nodes_up_and_normal(nodes=new_nodes)
+        # TODO: rewrite up and normal nodes
+        # skip all zero toke nodes for now
+        self.cluster.wait_for_nodes_up_and_normal(nodes=[node for node in new_nodes if not node._zero_token_node])
```

Comment:
those nodes are not shown on `nodetool status`  ? we are we waiting for them ?

if we shouldn't be waiting for any reason, we might want an argument in this function, skipping, instead of logic like that

Reply:
> those nodes are not shown on nodetool status ? we are we waiting for them ?

this is WIP: https://github.com/scylladb/scylladb/pull/20909

Reply:
Added workaround to use cql request